apps/studio/components/ui/ConfirmationModal.tsx

```diff
@@ -67,7 +67,7 @@ const ConfirmationModal = ({
             disabled={loading || buttonDisabled}
             onClick={onConfirm}
           >
-            {loading ? buttonLoadingLabel : buttonLabel}
+            {loading ? buttonLoadingLabel || buttonLabel : buttonLabel}
```

Comment:
Please confirm that this is actually the same thing (not entirely necessary, but who doesn't like a bit of boolean logic?
```suggestion
            {(loading && buttonLoadingLabel) || buttonLabel}
```

Reply:
^ oh i added this so that if its loading, but we dont have buttonLoadingLabel, use buttonLabel as the fallback

Reply:
I *think* that's what my one should do too, but since it's clearly more confusing (😅), let's go with the ternary one

Reply:
ohhhh actually yeah yours is fine tbh - removes the duplicate of calling buttonLabel too haha